pkg/apis/kops/v1alpha2/instancegroup.go

```diff
@@ -49,6 +49,17 @@ type InstanceGroupList struct {
 // InstanceGroupRole string describes the roles of the nodes in this InstanceGroup (master or nodes)
 type InstanceGroupRole string
 
+const (
+	// InstanceGroupRoleMaster is a master role
+	InstanceGroupRoleMaster InstanceGroupRole = "Master"
+	// InstanceGroupRoleNode is a node role
+	InstanceGroupRoleNode InstanceGroupRole = "Node"
+	// InstanceGroupRoleBastion is a bastion role
+	InstanceGroupRoleBastion InstanceGroupRole = "Bastion"
+	// InstanceGroupRoleAPIServer is an API server role
+	InstanceGroupRoleAPIServer InstanceGroupRole = "APIServer"
+)
+
```

Comment:
What are these used for?

Reply:
BTW I don't _really_ care too much here, if we don't want to duplicate the constants I can just import from the unversioned package.  I do think this is more correct though (even though it is more tedious).

Reply:
There doesn't appear to be a need to have versioned constants. The versioned structs are only ever accessed through serialization/deserialization and the apimachinery conversion code.

Reply:
OK, I'll remove.

The side-question it opens up is whether we should be using the unversioned types.  For example, here we're parsing the output from `kops get cluster`, which is naturally versioned.  If we do need to convert it, should we convert to a versioned type?  (i.e. is the unversioned type actually just apimachinery internals "leaking").  But ... a topic for another day!